roles/backup/tasks/main.yml

```diff
@@ -0,0 +1,56 @@
+- name: Check if epel file exists and register var
```

Comment:
This task should be a better check for the epel repo being installed - check if the repo is available rather than just for a file on the OS.

Reply:
Changed this to check for /etc/yum.repos.d/epel.repo - best way to check as yum module list function won't work.